app/javascript/controllers/index.js

```diff
@@ -13,12 +13,12 @@ application.register("aos", AosController)
 import BookingFormPriceController from "./booking_form_price_controller"
 application.register("booking-form-price", BookingFormPriceController)
 
-import Carousel from 'stimulus-carousel'
-application.register('carousel', Carousel)
-
```

Comment:
Why did you remove this ?

Reply:
for some reason when did the `git pull origin master` my controller to hide the alerts went gone, then I did `rails stimulus:manifest:update` on my branch to get it going again

Reply:
Ok, the Carousel is not part of the manifest so when we update the manifest it's removed from the index.js
You need to add to back manually please